core/tests/ts-integration/tests/l1.test.ts

```diff
@@ -54,6 +87,7 @@ describe('Tests for L1 behavior', () => {
             alice.requestExecute({
                 contractAddress: counterContract.address,
                 calldata,
+                mintValue: isETHBasedChain ? ethers.BigNumber.from(0) : expectedL2Costs,
```

Comment:
So it is not zero for not Eth based chains. It is zero in case of Eth chains, as in Eth based chain test no mintValue was passed.

Reply:
I think what @kelemeno means is that `mintValue` semantics should be simply "deposit value". Whether it is passed as msg.value or a method argument should be decided by the SDK based on the base token.

Reply:
If the eth based chains could be 0 then I don't see why it could not also be 0 for custom base chains? 

What I meant that "`mintValue`" is the some of `l2GasFees` and the `l2Value`, and is equal to the total deposited value on L1. And yes, normally when depositing tokens the SDK decides.